soapui/src/main/java/com/eviware/soapui/support/editor/views/xml/source/XmlSourceEditorView.java

```diff
@@ -628,7 +630,15 @@ public void setEditable(boolean enabled) {
 
     @Override
     public int getSupportScoreForContentType(String contentType ) {
-        return contentType.toLowerCase().endsWith("xml")? 2 : 0;
+        if(contentType.endsWith("xml")){
+            return 2;
+        }
+        Optional<String> any = Arrays.stream(contentType.split(";"))

```

Comment:
If there is other stuff like charset or boundard or both, we get rid of that and do the check again.